test/TestDNSSEC.js

```diff
@@ -66,18 +67,12 @@ async function verifyFailedSubmission(instance, data, sig, proof) {
     }
 
     try{
-        var tx = await instance.submitRRSet(data, sig, proof);
-    }
-    catch(error){
-        // @TODO use: https://github.com/ensdomains/root/blob/master/test/helpers/Utils.js#L8
-        // Assert ganache revert exception
-        assert.equal(error.message, 'Returned error: VM Exception while processing transaction: revert');
+        await instance.submitRRSet(data, sig, proof);
+    } catch(error){
```

Comment:
This won't work as desired - the test will still pass if no exception is thrown.

Reply:
No, due to line 75